tools/data/hmdb51/download_annotations.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/bash env
```

Comment:
this line looks suspicious. isn't it `#!/usr/bin/env bash`?

Reply:
Change it to `#!/usr/bin/env bash`. Same changes in other scripts are made in https://github.com/open-mmlab/mmaction2/pull/62